lib/collab.php

```diff
@@ -0,0 +1,43 @@
+<?php
```

Comment:
This file should move within the experimental folder, otherwise it will be backported to core.

Reply:
Moved within Experimental folder